src/fft.rg

```diff
@@ -126,16 +138,12 @@ function fft.generate_fft_interface(itype, dtype)
   local DEFAULT_TUNABLE_GLOBAL_OMPS = 9
   local DEFAULT_TUNABLE_GLOBAL_PYS = 10
 
+
   __demand(__inline)
   task iface.get_tunable(tunable_id : int)
     var f = c.legion_runtime_select_tunable_value(__runtime(), __context(), tunable_id, 0, 0)
     var n = __future(int64, f)
-
-    -- FIXME (Elliott): I thought Regent was supposed to copy on
-    -- assignment, but that seems not to happen here, so this would
-    -- result in a double destroy if we free here.
-
-    -- c.legion_future_destroy(f)
+    -- c.legion_future_destroy(f) -- Comment from Elliott: I thought Regent was supposed to copy on assignment, but that seems not to happen here, so this would result in a double destroy if we free here.
```

Comment:
Seems like this is being reformatted for no purpose?

Reply:
To cover in my formatting scrub